tests/support_server/setup.pm

```diff
@@ -263,6 +261,7 @@ sub setup_xvnc_server {
     send_key 'ret';
     wait_serial('yast-remote-status-0', 90) || die "'yast remote' didn't finish";
     wait_still_screen 3;
+    script_run 'yast2 firewall services add zone=EXT service=service:vnc-server';
```

Comment:
why not `assert_script_run`? Should this be allowed to fail?

Reply:
fixed.

Verification run: 
- http://10.67.17.30/tests/249
- http://10.67.17.30/tests/250